### ISSUE TITLE
[Feature reguest] Add attribute for fixed space between flex lines when using wrap or wrap_reverse

### ISSUE THREAD

**Issue by jdkoren** [google/flexbox-layout#82]
It would be nice to be able to specify a fixed amount of space between flex lines when using `flexWrap="wrap"` or `flexWrap="wrap_reverse"`. This would handle the case when the FlexBoxLayout's width or height is `wrap_content` (where `alignContent="space_between"` and 'alignContent="space_around"` are not really applicable).

**Comment by lii2** [comment]
I think you can do this using alignItems.

**Comment by jdkoren** [comment]
@lii2 I do not think so. My use case involves a `FlexBoxLayout` with `android:layout_height="wrap_content"` and `flexDirection="row"`. All the children are the same height, and the number of children can vary. If the children wrap to a new line, I would like to have space between the lines.

My understanding of the `alignItems` attribute is that it aligns children along the cross axis (which in this case is the vertical axis) _within the line_. With the constraints above, each line should be exactly the height of the children, which makes none of the `alignItems` values applicable.

IN any case, I see nothing about using fixed spaces anywhere among any of the justify/align attributes. I only see ways to distribute extra space or to align items within that space, but none of that applies when the layout is intended to be only large enough to contain its children.

**Comment by thagikura** [comment]
@jdkoren you can put fixed size of spaces by putting `layout_marginTop` (or bottom) to each child just like other ViewGroups.

**Comment by lii2** [comment]
Adding a margin or padding to the top and bottom of your child views would force there to be a line between them. Example: 
```
layout_marginTop="10px"
```

**Comment by jdkoren** [comment]
@thagikura @lii2 That's not really a workable solution. That margin will be present even if there's a single line, which will make the container larger than it's supposed to be. Again, I cannot know in advance which children will wrap and so I cannot put margins only on some children.

**Comment by jdkoren** [comment]
In fact, if I put `layout_marginBottom` or `layout_marginTop` on all children, there will always be unwanted space.
```
marginBottom, one line:
______________________
| child_1 child_2    |
|                    | <- unwanted
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
marginBottom, 2+ lines:
______________________
| child_1 child_2    |
|                    |
| child_3 (wrapped)  |
|                    | <- unwanted
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾

marginTop, one line:
______________________
|                    | <- unwanted
| child_1 child_2    |
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
marginTop, 2+ lines:
______________________
|                    | <- unwanted
| child_1 child_2    |
|                    |
| child_3 (wrapped)  |
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
```

**Comment by thagikura** [comment]
Thanks for the clarification. Ok that effect to have fixed size of spaces between flex lines without extra space at the bottom (or top) is indeed not feasible at this moment.

Adding a new attribute similar to the divider in the LinearLayout may make sense to be added.

I'll keep this issue open to revisit this later.

**Comment by thagikura** [comment]
This should be fixed by #100 .

I'll release the updated library anytime soon.
